docs/modules/reference/pages/provisioning/detectors.adoc

```diff
@@ -5,6 +5,45 @@ This section describes all available built-in xref:operation:provisioning/introd
 You can configure them to allow complex detection to meet the needs of your environment.
 For information on how to extend the functionality of these detectors, see the xref:development:development.adoc#development[Development section] to contribute to the project.
 
+[[ref-detector-provisioning-common-parameters]]
+== Common configuration parameters
+
+Application- or device-specific detectors are based on a generic API that provides common configuration parameters.
+These minimum configuration parameters are available in all detectors and describe the behavior for timeouts, retries, and so on.
+
+.Optional common configuration parameters
+[options="header"]
+[cols="1,3,2"]
+|===
+| Parameter
+| Description
+| Default value
+
+| ipMatch
+| IP address to match.
+Use filters to match single addresses or address ranges.
+Substrings are matched by default (see note below this table for information on regular expression matching). +
+Note: ipMatch is a required parameter for xref:reference:provisioning/detectors/LoopDetector.adoc[LoopDetector].
```

Comment:
The notes may not be needed here if the specific detector has information stating it is required.

Reply:
I'll go ahead and update this! The files you mentioned already have the appropriate disclaimers, so no updates will need to be made there. :)

Reply:
Awesome. I forgot that!